test-validator/src/lib.rs

```diff
@@ -107,9 +107,14 @@ pub struct TestValidatorGenesis {
     pub max_genesis_archive_unpacked_size: Option<u64>,
     pub accountsdb_plugin_config_files: Option<Vec<PathBuf>>,
     pub accounts_db_caching_enabled: bool,
+    inactivate_feature_list: Option<Vec<Pubkey>>,
 }
 
 impl TestValidatorGenesis {
+    pub fn inactivate_features(&mut self, inactivate_list: Vec<Pubkey>) -> &mut Self {
```

Comment:
I think I prefer "deactivate" over "inactivate", to better align with existing terminology in `sdk/src/feature_set.rs`.

Let's also pass in a slice instead of a Vec and extend the list in `self`, so that `deactivate_features()` can be called multiple times if the caller desires

```suggestion
    pub fn deactivate_features(&mut self, inactivate_list: &[Pubkey]) -> &mut Self {
```

Reply:
Changed to reflect comment in fb8b5b5